recipe/fix-path-error-on-win.patch

```diff
@@ -0,0 +1,20 @@
+This fixes an error where the path '/' is attempted to be converted from a
+relative to absolute path on windows, resulting in a ValueError.
+
+Note that this patch stops setup.py ... bdist_wheel --universal from being
```

Comment:
universal or not won't affect noarch. This is just the generated wheel, which gets installed. We don't distribute wheels.

Reply:
WAit, I see what you mean.

Reply:
ok, thanks. But I was wondering if conda-build uses --universal to build noarch packages. Because perhaps the site-packages paths in the wheel don't contain python version in this case. Is that not the case?

Reply:
nope, this is just the installation path upon installation, not any paths contained in the wheel

Reply:
ah ok, I see what you mean